src/plugins/index_pattern_field_editor/public/open_editor.tsx

```diff
@@ -132,6 +132,9 @@ export const getFieldEditorOpener = ({
         size: 'l',
         ownFocus: true,
         hideCloseButton: true,
+        'aria-label': i18n.translate('indexPatternFieldEditor.flyoutAriaLabel', {
+          defaultMessage: 'Create or edit field',
```

Comment:
Adding the name of the field will add a little more helpful context to this menu

```suggestion
          defaultMessage: 'Create or edit {fieldName} field',
          values: {{ fieldName }},
```

Reply:
Good idea. I decided to have 2 different aria-labels: on for creating and one for editing.